ui/src/App.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
```

Comment:
Change double quotes in this file into single quotes as well.

Reply:
Done.